docs/developers/testing.md

```diff
@@ -111,3 +111,118 @@ If you added a unit test file, you must add your new test to the `unit_tests_lis
     // ... other tests ...
 ]
 ```
+
+## iRODS Testing Environment
+
+The iRODS Testing Environment is a Docker Compose-based project written in Python that is used by the Consortium to test the iRODS server and its plugins. The Testing Environment is driven by a series of scripts to run the tests on a number of different platforms with various database backends and other configuration options. This is the primary tool used for day-to-day testing for iRODS developers as well as for verification in releasing the iRODS server. In addition to running tests, the Compose projects can be useful for manually testing things and quickly deploying specific environments for reproducing issues or developing a proof of concept.
+
+This documentation will briefly describe a few useful ways to employ the iRODS Testing Environment. The README in the project's GitHub repository provides extensive details on the purpose and usage for the various scripts: [https://github.com/irods/irods_testing_environment/blob/main/README.md](https://github.com/irods/irods_testing_environment/blob/main/README.md) Each script available in the project also has help text available with the `--help` option which describes the available options and their purposes.
+
+### Getting started
+
+In order to configure your environment to use the iRODS Testing Environment, using a virtual environment is highly recommended. Here's how to set up a virtual environment and install the required Python packages:
+```
+virtualenv -p python3 ~/irods_testing_environment
```

Comment:
According to the python docs, `python -m venv` is the recommended approach for creating virtual environments.

https://docs.python.org/3/library/venv.html#creating-virtual-environments

Reply:
Done